### PR TITLE
add missing helmRepository option for input

### PR DESCRIPTION
**Pull request by mandelsoft**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release Notes**:

```feature user

```

### CODE REVIEW

**Review by morri-son — CHANGES_REQUESTED** [review]
I guess there are typos in the method name, or? https://github.com/open-component-model/ocm/pull/462/files#diff-ed03f9f3b9f081bd7ec3d81c4a2e6091830405427983a0589ee6fa4f9c4b1a13R15 Should be HelmRepositoryOption instead

**Comment by mandelsoft** [comment]
yes, thanks, fixed....

**Review by morri-son — CHANGES_REQUESTED** [review]
another one :)

https://github.com/open-component-model/ocm/pull/462/files#diff-ed03f9f3b9f081bd7ec3d81c4a2e6091830405427983a0589ee6fa4f9c4b1a13R15

**Comment by mandelsoft** [comment]
That is strange, on the mandelsoft/helm branch is a different commit than in the PR.

**Comment by mandelsoft** [comment]
I amended the commit an repuhed, not it gets the correct one (and also shows the correct old one, which was not visible in the UI)

File: docs/reference/ocm_add_resource-configuration.md [hunk 1]
```diff
@@ -313,6 +314,13 @@ with the field <code>type</code> in the <code>input</code> field:
     Basically, it is a good practice to use the component version for local resources
     This can be achieved by using templating for this attribute in the resource file.
   
+  - **<code>helmRepository</code>** *string*
+  
+    This OPTIONAL property can be set, if the helm chart should be loaded from 
+    a helm repository instead of the local filesystem. It describes
+    the base URL of the chart repository. If specified, the <code>path</code> field
+    must describe the name of the chart in the chart repository.
```
Comment (reshnm) [inline docs/reference/ocm_add_resource-configuration.md:322]: I think it should also describe, that the version field is required when `helmRepository` is set.

**Review by reshnm — CHANGES_REQUESTED** [review]

File: docs/reference/ocm_add_resource-configuration.md (hunk 1 above)
Comment (mandelsoft) [reply docs/reference/ocm_add_resource-configuration.md]: yes, I added this.

**Review by reshnm — APPROVED** [review]
/lgtm

**Review by morri-son — APPROVED** [review]

**Pull request merged**